doc/conf/Doxyfile.in

```diff
@@ -1,27 +1,29 @@
 QUIET                  = YES
 
 # General project info
-PROJECT_NAME           = "Ginkgo"
-PROJECT_NUMBER         = "@PROJECT_VERSION_TAG@ @PROJECT_VERSION@"
-PROJECT_BRIEF          = "Ginkgo is a numerical linear algebra library targeting manycore architectures."
+PROJECT_NAME           = "@PROJECT_NAME@" 
+PROJECT_NUMBER         = "Generated from @GINKGO_GIT_BRANCH@ branch based on @PROJECT_VERSION_TAG@. Ginkgo version @PROJECT_VERSION@ "
+PROJECT_BRIEF          = "@PROJECT_DESCRIPTION@" 
 INLINE_INHERITED_MEMB  = NO
 
 # Paths and in/exclusion patterns
-INPUT                  = @DIR_BASE@/core @DIR_BASE@/omp @DIR_BASE@/cuda @DIR_BASE@/reference @MAINPAGE@
-INCLUDE_PATH           = @DIR_BASE@/include @DIR_BASE@
+#INPUT                  = See the specific doxy files for usr and dev.  
```

Comment:
Commented code

Reply:
It was meant to be an explanation for anyone searching where the INPUT variable is defined.

Reply:
Ah I see, maybe you could change this into a sentence then so that it catches more the eye, e.g.:
```
The INPUT variable is defined in the specific doxy files for usr and dev.
```